ZAPD/Main.cpp

```diff
@@ -37,22 +37,22 @@ void BuildAssetModelIntermediette(const std::string& outPath);
 void BuildAssetAnimationIntermediette(const std::string& animPath, const std::string& outPath);
 
 #if !defined(_MSC_VER) && !defined(__CYGWIN__)
-void ErrorHandler(int sig)
+void ErrorHandler(int32_t sig)
```

Comment:
A function passed to `signal` is expected to have one parameter, and it should be of type `int`. The signature should be:
```c
 typedef void (*sighandler_t)(int);
```

```suggestion
void ErrorHandler(int sig)
```